Iced/Intel/EncoderInternal/OpHandlers.cs

```diff
@@ -370,6 +373,9 @@ static class LegacyOps {
 			new OpReg(Register.R15),
 			new OpReg(Register.ST0),
 			new OpRegSTi(),
+			new OpRegOpCodeByte(Register.AX, Register.DI),
+			new OpRegOpCodeByte(Register.EAX, Register.EDI),
+			new OpRegOpCodeByte(Register.RAX, Register.RDI),
```

Comment:
You'll probably need three more handlers for the other instructions since they can use all 16 regs.

Reply:
No, I realized you don't need it. The encoder verifies that only the low 8 regs are used if it's 16 or 32-bit mode, so you can change the last reg to R15W/R15D/R15 here.